dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -937,7 +937,7 @@
           "$ref": "#/definitions/CustomHealthcareEntitiesDocumentResult"
         },
         {
-          "$ref": "#/definitions/DocumentDetectedLanguageString"
+          "$ref": "#/definitions/DocumentDetectedLanguage"
```

Comment:
Can we also update `HealthcareEntitiesDocumentResultWithDocumentDetectedLanguage` to use obj instead of string?

Reply:
@aurghob is already working on this